### PR TITLE
MITRE ATT&CK - ignore revoked indicators when checking names of attack patterns

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-12207

## Description
The command did not ignore revoked indicators etracting IDs that have been revoked by MITRE.

### CODE REVIEW

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_44.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### MITRE ATT&CK
+Fixed an issue with the "get_mitre_value_from_id" command not ignoring revoked indicators.
```
Comment (MLainer1) [inline Packs/FeedMitreAttackv2/ReleaseNotes/1_1_44.md:5]:
```suggestion
Fixed an issue where the "get_mitre_value_from_id" command did not ignore revoked indicators.
```

**Review by MLainer1 — APPROVED** [review]
🆒

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_44.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### MITRE ATT&CK
+Fixed an issue where the "get_mitre_value_from_id" command did not ignore revoked indicators.
```
Comment (ShirleyDenkberg) [inline Packs/FeedMitreAttackv2/ReleaseNotes/1_1_44.md:5]: Do you need to include updated Docker image?

**Comment by ShirleyDenkberg** [comment]
@MLainer1 Doc review completed.

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_44.md (hunk 2 above)
Comment (Ni-Knight) [reply Packs/FeedMitreAttackv2/ReleaseNotes/1_1_44.md]: Someone just updated it the commit before :)

**Pull request merged**